app/views/admin/_admin_nav.html.erb

```diff
@@ -29,8 +29,9 @@
   <li>
     <%= span_if_current ts("Known Issues", key: "header"), known_issues_path %>
   </li>
-  <li>
-    <%= span_if_current ts("Wrangling Guidelines", key: "header"),
-                        wrangling_guidelines_path %>
-  </li>
+  <% if policy(:wrangling).new? %>
+    <li>
+      <%= span_if_current t("admin.admin_nav.wrangling_guidelines"), wrangling_guidelines_path %>
```

Comment:
This can be shortened by using a relative key:
```suggestion
      <%= span_if_current t(".wrangling_guidelines"), wrangling_guidelines_path %>
```